src/Service_Providers/Base.php

```diff
@@ -66,8 +66,13 @@ public function admin_notices() {
 		$links = array();
 
 		foreach ( $this->get_requisite_plugins() as $plugin ) {
-			$links[] = sprintf( '<a href="%1$s" class="thickbox" title="%2$s">%3$s</a>', esc_attr( $plugin['thickbox_url'] ),
-				esc_attr( $plugin['short_name'] ), esc_html( $plugin['short_name'] ) );
+			if ( isset( $plugin['external_download'] ) && true === $plugin['external_download'] ) {
+				$links[] = sprintf( '<a href="%1$s" title="%2$s" target="_blank">%3$s</a>', esc_attr( $plugin['thickbox_url'] ),
+					esc_attr( $plugin['short_name'] ), esc_html( $plugin['short_name'] ) );
+			} else {
+				$links[] = sprintf( '<a href="%1$s" class="thickbox" title="%2$s">%3$s</a>', esc_attr( $plugin['thickbox_url'] ),
+					esc_attr( $plugin['short_name'] ), esc_html( $plugin['short_name'] ) );
```

Comment:
you can break this and the above `sprintf()` into multiple lines -- i think you should